src/lib.rs

```diff
@@ -312,6 +312,37 @@ impl ClamdClientBuilder {
     }
 }
 
+pub enum ScanResult {
+    Benign,
+    Malignent { infection_types: Vec<String> },
+}
+
+impl ScanResult {
+    pub(crate) fn from_output(out: &str) -> Result<Self> {
+        println!("{}", out);
+        if out.contains("OK") {
```

Comment:
This could be bad: imagine if a virus signature contains this string "OK": the result will return as benign even though it is not. Why not keep that part as it was (check whether out == "OK") ?

Reply:
It seems unlikely to me that this would happen, but not impossible.

I've done it this way because when you create an `IDSESSION`, you get the number of the operation as a prefix, like so : `1: OK` or `2: FOUND: ...`.

I'll try to parse the output more intelligently

Reply:
That should do it, lmk what you think